test-apps/display-test-app/src/common/DtaConfiguration.ts

```diff
@@ -141,6 +143,9 @@ export const getConfig = (): DtaConfiguration => {
   if (undefined !== process.env.IMJS_BING_MAPS_KEY)
     configuration.bingMapsKey = process.env.IMJS_BING_MAPS_KEY;
 
+  if (undefined !== process.env.IMJS_GOOGLE_MAPS_KEY)
```

Comment:
Could you add these environment variables to the DTA README?

Reply:
done

Reply:
Thanks. Did you push it? I don't see it in the diff and the only new commits I see are merging master in

Reply:
push got blocked, should be good now